### PR TITLE
<PopoverMenu.MenuItem/> <ListItemAction/> - add subtitle property

### PR DESCRIPTION
**Pull request by elad-barack**
# 🔦 Summary

Added a "subtitle" option for a PopoverMenu.MenuItem & ListItemAction(The "PopoverMenu.MenuItem" component is using it, so had to add the support to ListItemAction first)

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain) - ListIconAction/README.md , PopoverMenu/compound.md
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by EgoziE** [comment]
Hey @elad-barack 
Who did you contact with from our team about this change?
Was this change approved by someone from our UX?

**Comment by elad-barack** [comment]
Hi, I did aproove the design (of the feature, not the PR yet) with @ayeletr and Yael Raz. Didn't speak to someone from the dev team yet. Can we do it on Sunday?

File: src/PopoverMenu/PopoverMenu.js [hunk 1]
```diff
@@ -195,6 +195,7 @@ class PopoverMenu extends React.PureComponent {
           dataHook: child.props.dataHook,
           prefixIcon: child.props.prefixIcon,
           disabled: child.props.disabled,
+          subtitle: child.props.subtitle,
```
Comment (EgoziE) [inline src/PopoverMenu/PopoverMenu.js:198]: Please add it to PopoverMenuItemProps in the relevant d.ts file and type test (tsx file)

**Review by EgoziE — CHANGES_REQUESTED** [review]

File: src/PopoverMenu/PopoverMenu.js (hunk 1 above)
Comment (elad-barack) [reply src/PopoverMenu/PopoverMenu.js:198]: Done

**Review by EgoziE — APPROVED** [review]
Looks good, is it still waiting for UX review?

**Comment by elad-barack** [comment]
@EgoziE Yes, accidentally asked you to approve. But It's always better to get one step closer :)

**Comment by ayeletr** [comment]
Hi, @elad-barack 
small comments:

**1.** on this example:
image
please add icons to each line. I don't want to show unreal examples.

**2.** When the content is very long and there's ellipsis, there's no padding on the right side. need to have 24px padding there.
image
(I see it also happen for the title, maybe it something that was exist)

Thanks!

**Pull request merged**